src/lightbox/Lightbox.js

```diff
@@ -95,9 +95,9 @@ export default function Lightbox(props: Props) {
       </SlideAnimationView>
       <SlideAnimationView
         property="translateY"
-        style={[styles.overlay, { width, bottom: height - 44 }]}
-        from={height}
-        to={height - 44}
+        style={[styles.overlay, { width: windowWidth, bottom: windowHeight - 44 }]}
+        from={windowHeight}
+        to={windowHeight - 44}
```

Comment:
I also notice, thanks to this cleanup calling my attention to it, that this longstanding bit of code is all unnecessarily complicated -- if you subtract `windowHeight` from all three of these vertical values, it gets a lot simpler. Maybe simpler still if you subtract `windowHeight - 44`, so that it's:
* `bottom: 0`
* `from={44}`
* `to={0}`

But that might all be made moot by the changes you're [already working on](https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/.60View.60's.20.60onLayout.60.20prop/near/1110028) to replace this `translateY` animation entirely.

Reply:
I've got a commit that does exactly that simplification; about to send the PR. 😄